### ISSUE TITLE
Schedule router reboot instead of wifi on/off

### ISSUE THREAD

**Issue by dhcmega** [jazzran/ZTEF680_Wifi_En#3]
Hi
Thanks for sharing your script.
I'm familiar with PHP, but python seems pretty similar.
Do you think that the script will be suitable for adding a new function/method that will trigger a reboot of the router?
In may case, the router gets disconnected in time to time and I would like to schedule a midnight reboot (for when it happens and I'm abroad)
Another better alternative would be a network test and if it fails X times in a row, then reboot.
Thanks!

dhcmega

**Comment by jazzran** [comment]
Hi,
Thanks for the interest in the project.
I don't have available the Router anymore, so I can't get the url call that is needed to perform the reboot. If you are able to obtain it, I can upgrade the script with this extra method.
Thanks in advance!

**Comment by dhcmega** [comment]
Hi, thanks for answering
I have just changed my provider. Now I have a Sagemcom FAST3890V3. So I also don't have access to a ZTE any more.
Thanks!

**Comment by macmonty** [comment]
> Hi, Thanks for the interest in the project. I don't have available the Router anymore, so I can't get the url call that is needed to perform the reboot. If you are able to obtain it, I can upgrade the script with this extra method. Thanks in advance!

Yo estoy interesado en esa modificación, ¿que url necesitas? ¿la que se aplica con el botón?

Muchas gracias!

**Comment by jazzran** [comment]
Sí, sería la url que se llama cuando se hace click sobre el boton de  reboot.
